website/blog/2019-09-23-linux-foundation.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Presto now hosted under the Linux Foundation
+author: Ariel Weisberg
+authorURL: https://www.linkedin.com/in/ariel-weisberg-a5b6899
+authorFBID: 100006183888902
+---
+
+We are happy to announce that today we have launched the Presto Foundation, a non-profit organization under the umbrella of the Linux Foundation, together with our partners Alibaba, Twitter, and Uber!
+
+Being hosted by the Linux Foundation opens up the Presto community to a broader ecosystem of users and contributors. The Presto Foundation's open and neutral governance enables the community to influence Presto's future, which will also make it more attractive to developers. Together, we will raise Presto's performance, scalability, and reliability to new heights that could never have been reached alone.
```

Comment:
s/Being hosted/Hosting

Link to the "open and neutral governance", if it exists.

Reply:
Unfortunately the bylaws are not visible anywhere yet. I did ask Michael Cheng about it and he said that needs to be figured out. 

The "open and neutral" wording is from the LF press release.